book/src/leader-rotation.md

```diff
@@ -3,17 +3,76 @@
 At any given moment, a cluster expects only one fullnode to produce ledger
 entries. By having only one leader at a time, all validators are able to replay
 identical copies of the ledger. The drawback of only one leader at a time,
-however, is that a malicious leader is cabable of censoring votes and
+however, is that a malicious leader is capable of censoring votes and
 transactions. Since censoring cannot be distinguished from the network dropping
 packets, the cluster cannot simply elect a single node to hold the leader role
-indefinitely. Instead, the cluster minimizes the influence of a malcioius
+indefinitely. Instead, the cluster minimizes the influence of a malicious
 leader by rotating which node takes the lead.
 
 Each validator selects the expected leader using the same algorithm, described
 below. When the validator receives a new signed ledger entry, it can be certain
 that entry was produced by the expected leader.
 
-## Leader Schedule Generation
+## Leader Schedule Rotation
+
+A block's slot is valid if it is generated by the leader computed from a Leader
+Schedule in the block that crossed the epoch slot boundary at least 1 epoch ago.
+
+Leader schedule is generated every epoch when a fork crosses the epoch boundary.
+In practice the schedule only needs to be generated when the root fork crosses
+the epoch boundary.
+
+The LS is active at the next epoch.  The epoch should be long enough such that
```

Comment:
This acronym isn't defined above. There's no precedent for 2-letter acronyms in the codebase (if we want to create one here, we should first get rid of all the 2-letter variables like tx and ix in the codebase). I'd recommend just expanding it and writing leader scheduler everywhere. Maybe sometime later we'll think up a better name for that object. It's currently the only struct with a "worker" name, reminiscent of the old "Accountant" that was renamed "Bank".